go.mod

```diff
@@ -19,11 +19,11 @@ require (
 	github.com/gorilla/websocket v1.5.1
 	github.com/pborman/uuid v1.2.1
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.16.0
+	github.com/prometheus/client_golang v1.19.1
 	github.com/rancher/apiserver v0.0.0-20240708202538-39a6f2535146
 	github.com/rancher/dynamiclistener v0.6.0-rc2
 	github.com/rancher/kubernetes-provider-detector v0.1.5
-	github.com/rancher/lasso v0.0.0-20240923125127-ae858d002589
+	github.com/rancher/lasso v0.0.0-20241202185148-04649f379358
```

Comment:
(See the other comment I left for context)

This makes me wonder, is there anything forcing us to keep the sql cache in lasso itself? It feels like it would be much easier to move most of it to steve directly, that way no longer need to deal with bumping lasso and changes to Vai would only need to be done in two places (steve, + bump to Rancher) instead of three (lasso + steve + bump to rancher)

Thoughts? (cc: @ericpromislow since you've worked on this a lot too)

Reply:
> This makes me wonder, is there anything forcing us to keep the sql cache in lasso itself? It feels like it would be much easier to move most of it to steve directly, that way no longer need to deal with bumping lasso and changes to Vai would only need to be done in two places (steve, + bump to Rancher) instead of three (lasso + steve + bump to rancher)

I am not aware of any such constraints. It was a design choice to keep some of the complexity in lasso, under the premise it could have been more reusable there. FMPOV, until and unless we have an actual use case, logic can very well be kept all in one place.

Reply:
Anyway we take this discussion, I believe it should not block this PR, right?

Reply:
Yes no need to block this.